templates/serverpod_templates/projectname_server_upgrade/Dockerfile

```diff
@@ -21,9 +21,10 @@ COPY --from=build /runtime/ /
 COPY --from=build /app/bin/main /app/bin/main
 COPY --from=build /app/config/ config/
 COPY --from=build /app/web/ web/
+COPY --from=build /app/migrations/ migrations/
 
 EXPOSE 8080
 EXPOSE 8081
 EXPOSE 8082
 
-CMD app/bin/main --mode $runmode --server-id $serverid --logging $logging --role $role
+CMD app/bin/main --mode $runmode --server-id $serverid --logging $logging --role $role --apply-migrations
```

Comment:
Question: Applying migrations is often a step that the developer wants to have control over. Imagine starting multiple containers in one go, then we probably don't want all of them to apply the migrations.

Therefore we probably don't want them to always be applied but make it possible to specify them if needed.

I think we could utilize a docker [ENTRYPOINT](https://docs.docker.com/reference/dockerfile/#entrypoint) to achieve this. Where a resonable default is provided as the `CMD` but all flags can be overridden from the ENTRYPOINT.